### DISCUSSION TITLE
How to delete a journal?

### DISCUSSION THREAD

**Discussion by mifas** [pajoma/vscode-journal#117] [General]
**Edited**
Would like delete specific journal pages.

**Comment by pajoma** [comment]
Can you elaborate? Delete the whole all journal files, delete the extension, delete specific journal pages?

**Comment by mifas** [comment]
> Can you elaborate? Delete the whole all journal files, delete the extension, delete specific journal pages?

Updated the description

**Comment by mameen** [comment]
> > Can you elaborate? Delete the whole all journal files, delete the extension, delete specific journal pages?
> 
> Updated the description

I think if you sent a screen-shoot  we canhelp you better..

**Comment by pajoma** [comment]
If you use the Command "Open the Journal", you see all your journal files in the Explorer and can move or delete anything you want. 

If you want to delete a file currently active in the editor, maybe you try something like the [File Bunny Extension](https://marketplace.visualstudio.com/items?itemName=robole.file-bunny)
